### ISSUE TITLE
attention mask

### ISSUE THREAD

**Issue by Yueeeeeeee** [GT-SALT/MixText#26]
I want to ask in mixtext.py, why use torch.ones to define the attention_mask2 and torch.zeros to define token_type_ids2 instead of using the real attention mask and token types. Also, in the hidden representations, after the hidden representations are mixed, the attention_mask is used without mixing attention_mask2, can you explain the reason for that? Thank you!

**Comment by jiaaoc** [comment]
I did not use attention_mask at that work. But you could add it for experiments.
